jinja2/filters.py

```diff
@@ -51,21 +51,24 @@ def environmentfilter(f):
     return f
 
 
-def make_attrgetter(environment, attribute):
+def make_attrgetter(environment, attribute, lowercase=False):
```

Comment:
Does this really fit in this function? I'd rather have the option to pass an arbitrary `postprocess` callable that'll be applied before returning the attribute's value.